setup.cfg

```diff
@@ -50,45 +46,33 @@ keywords =
     wsgi
 
 [options]
-use_scm_version = True
-python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
 packages = find:
 include_package_data = True
+python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
 
-# These are required during `setup.py` run:
 setup_requires =
     setuptools_scm>=1.15.0
     setuptools_scm_git_archive>=1.0
 
-# These are required in actual runtime:
 install_requires =
     backports.functools_lru_cache
     six>=1.11.0
     more_itertools>=2.6
 
 [options.extras_require]
-docs =
+testing =
```

Comment:
Why not keep alphabetic ordering?

Reply:
I use semantic ordering - placing the more meaningful extras at the beginning, with the skeleton ones first and project-specific extras after. I don't feel strongly about the ordering, though I don't see a lot of value in maintaining alphabetic ordering. Other projects have different ordering preferences (such as shorter names first). For me, stability and consistency across projects is more important.